<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_meanMin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_meanMax">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Mean</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Mean</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_varMin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Variance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_varMax">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Variance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode per Class</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_modePerClass">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample per Mode</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_samplePerMode">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>700</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Process</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="defaultValueButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Default Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Generate Points</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Neural Network Hyperparameters</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nbOfEpoch">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>480</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nb of Epoch</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lr">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Learning Rate</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxActFunc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>620</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>heaviside</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>logistic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>tanh</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sign</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>relu</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>leaky relu</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Activation Function</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_betaValue">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>620</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_betaValue">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>600</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BÃªta Value</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_variableLr">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Variable Learning Rate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_maxLr">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>560</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_maxLr">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max lr</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_minLr">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min lr</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_minLr">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nbNeuronsInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Neurons in InputLayer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nbNeuronsHidden">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Neurons in HiddenLayer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
